portal-web/test/functional/com/liferay/portalweb/macros/MyTestFile.macro

```diff
@@ -0,0 +1,12 @@
+definition {
+    
+    macro assertConfiguration {
+        AssertVisible(locator1 = "MyTestFile#CONFIGURATION");
+    }
+    macro clickControPanel {
+        Click(locator1 = "MyTestFile#BUTTON_CONTROL_PANEL");
+    }
+    macro clickMenu {
+        Click(locator1 = "MyTestFile#BUTTON_MENU");
+    }
```

Comment:
You could have these two macros together and called "openControlPanel", something like that